app/adapters/base.js

```diff
@@ -7,6 +7,9 @@ import {inject as service} from '@ember/service';
 export default RESTAdapter.extend(DataAdapterMixin, AjaxServiceSupport, {
     host: window.location.origin,
     namespace: ghostPaths().apiRoot.slice(1),
+    xhrFields: {
+        withCredentials: true
```

Comment:
I'm pretty sure we need this

Reply:
Yeah. 🤔 We could set it only on the login request as that's what's setting the cookie but I guess that limits later separation of API and admin because when the origins are different no cookies will be sent without the `withCredentials` flag.

One thing I would say is that the flag probably belongs in the `ajax` service instead. Ember Data uses the ajax service under the hood via the [AjaxServiceSupport mixin](https://github.com/TryGhost/Ghost-Admin/blob/master/app/adapters/base.js#L1) but we also use the ajax service directly for API calls (NB. the ajax service should only be used for calls to our API, for external calls we should be using `fetch`).

Reply:
> I guess that limits later separation of API and admin because when the origins are different no cookies will be sent without the withCredentials flag.

Yeah I think we should deffo support that from the get go - less headaches later.

> One thing I would say is that the flag probably belongs in the ajax service instead.

Makes sense :+1: